traduzindo i18n parte inicial (#217)
campuscode/rails-guides-pt-BR: pt-BR/i18n.md

**NÃO LEIA ESTE ARQUIVO NO GITHUB, OS GUIAS SÃO PUBLICADOS NO https://guiarails.com.br.**
**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**

API de Internacionalização do Rails (I18n)
=====================================

> Ruby I18n(abreviação de intercionalização) é um "gem" fornecida com o Ruby on Rails(a partir do Rails 2.2) que fornece uma estrutura extensível e fácil de usar para **traduzir sua aplicação para um único idioma personalizado** que não seja o inglês ou **para fornecer suporte em multi-linguagens** em sua aplicação.

Reviewer: ```suggestion
Ruby I18n(abreviação de internacionalização) é um *gem* fornecida com o Ruby on Rails (a partir do Rails 2.2) que fornece uma estrutura extensível e fácil de usar para **traduzir sua aplicação para um único idioma personalizado** que não seja o inglês ou **para fornecer suporte em multilinguagens** em sua aplicação.
```

> Ruby I18n(abreviação de intercionalização) é um "gem" fornecida com o Ruby on Rails(a partir do Rails 2.2) que fornece uma estrutura extensível e fácil de usar para **traduzir sua aplicação para um único idioma personalizado** que não seja o inglês ou **para fornecer suporte em multi-linguagens** em sua aplicação.

Reviewer 2: ```suggestion
Ruby I18n (abreviação de internacionalização) é uma *gem* fornecida com o Ruby on Rails (a partir do Rails 2.2) que fornece uma estrutura extensível e fácil de usar para **traduzir sua aplicação para um único idioma personalizado** que não seja o inglês ou **para fornecer suporte multilinguagens** em sua aplicação.
```

Complementei com as anotações da @erikacamposdesign :smiley:

> O processo de "internacionalização" geralmente significa abstrair todas as strings e outros bits específicos do código do idioma(como formatos de data ou moeda) de sua aplicação. O processo de "localização" significa fornecer traduções e formatos locais para esses bits.

Reviewer 2: ```suggestion
O processo de "internacionalização" geralmente significa abstrair todas as *strings* e outros *bits* específicos do código do idioma (como formatos de data ou moeda) de sua aplicação. O processo de "localização" significa fornecer traduções e formatos locais para esses *bits*.
```

> Portanto, no processo de intercionalização de sua aplicação Rails, você deve:

Reviewer: ```suggestion
Portanto, no processo de internacionalização de sua aplicação Rails, você deve:
```

> * Verifique se tem suporte para i18n.

Reviewer: ```suggestion
* Verificar se tem suporte para i18n.
```
A mudar do verbo é para dar continuidade à frase anterior

> * Informe ao Rails onde encontrar dicionários locais.

Reviewer: ```suggestion
* Informar ao Rails onde encontrar dicionários locais.
```

> * Informe ao Rails como definir, preservar e alternar as localidades.

Reviewer: ```suggestion
* Informar ao Rails como definir, preservar e alternar as localidades.
```

> No processo de _localizing_ de sua aplicação, você provavelmente fará os três passos abaixo:

Reviewer: ```suggestion
No processo de "localização" de sua aplicação, você provavelmente fará os três passos abaixo:
```

> * Substituir ou complementar a localidade padrão do Rails - por exemplo formatos de data e hora, nomes de meses, nomes de modelos do Active Record, etc.

Reviewer: ```suggestion
* Substituir ou complementar a localidade padrão do Rails - por exemplo formatos de data e hora, nomes de meses, nomes de modelos do *Active Record*, etc.
```

> * Abstrair strings em sua aplicação utilizando dicionários com chave - por exemplo mensagens flash, texto estático em suas visualizações etc.

Reviewer 2: ```suggestion
* Abstrair *strings* em sua aplicação utilizando dicionários com chave - por exemplo mensagens *flash*, texto estático em suas visualizações etc.
```

* Armazenar os dicionários resultantes em um lugar específico.

> Este documento o guiará pela API I18n e contém um tutorial sobre como internacionalizar um aplicativo Rails desde o início.

Reviewer 2: ```suggestion
Este documento o guiará pela API I18n e contém um tutorial sobre como internacionalizar uma aplicação Rails desde o início.
```

> * Como o I18n funciona no Ruby on Rail

Reviewer: ```suggestion
* Como o I18n funciona no Ruby on Rails
```

> * Como usar corretamente o I18n em um aplicativo RESTful de várias maneiras

Reviewer 2: ```suggestion
* Como usar corretamente o I18n em uma aplicação RESTful de várias maneiras
```

> * Como usar o I18n para traduzir erros do Active Record ou assuntos de email do Action Mailer

Reviewer: ```suggestion
* Como usar o I18n para traduzir erros do *Active Record* ou assuntos de email do *Action Mailer*
```
